common/lib/common-utils/api-report/common-utils.api.md

```diff
@@ -78,6 +73,183 @@ export interface IComparer<T> {
     min: T;
 }
 
+// @internal @deprecated
```

Comment:
All the new exports in common-utils are things that moved here from common-definitions and need to be exported so the existing exports don't complain. I'm surprised the package didn't complain before; I would have expected to see re-exports of those things to placate api-extractor.